src/App.js

```diff
@@ -14,6 +15,7 @@ function App() {
       <Routes>
         <Route path="/" element={<ArticlesList />}></Route>
         <Route path="/articles/:article_id" element={<SingleArticle />}></Route>
+        <Route path="/articles/:article_id/comments" element={<CommentsList />}></Route>
```

Comment:
I don't think this should be its own route - i think it would feel like a more natural user journey to see the comments directly under an article, or be able to expand a comment list without being taken to a new page.